pkg/dashboard/resource/frontendspec.go

```diff
@@ -94,13 +94,20 @@ func (fesr *frontendSpecResource) getFrontendSpec(request *http.Request) (*restf
 func (fesr *frontendSpecResource) getDefaultFunctionConfig() map[string]interface{} {
 	one := 1
 	defaultWorkerAvailabilityTimeoutMilliseconds := trigger.DefaultWorkerAvailabilityTimeoutMilliseconds
+	defaultHTTPTrigger := functionconfig.GetDefaultHTTPTrigger()
+	defaultHTTPTrigger.WorkerAvailabilityTimeoutMilliseconds = &defaultWorkerAvailabilityTimeoutMilliseconds
```

Comment:
Because this creates an import loop.
To get `trigger.DefaultWorkerAvailabilityTimeoutMilliseconds` I need to import `"github.com/nuclio/nuclio/pkg/processor/trigger"`. But `"github.com/nuclio/nuclio/pkg/processor/trigger"` imports `"github.com/nuclio/nuclio/pkg/functionconfig"`. So if I put this line in `functionconfig.GetDefaultHTTPTrigger` those packages will import each other, and this doesn't work.

In addition, I filled the default trigger with the parameters I got while creating a trigger in the ui and then inspecting it's spec. Meaning this isn't necessary for a default trigger, yet I didn't want to break UI functionality.

Reply:
I was afraid you were gonna say that